### PR TITLE
Add validation dashboard delete

### PR DESCRIPTION
**Pull request by callemand**
### Pull Request check-list

To ensure your Pull Request can be accepted as fast as possible, make sure to review and check all of these items:

- [x] If your changes affects code, did your write the tests?
- [x] Are tests passing? (`npm test` on both front/server)
- [x] Is the linter passing? (`npm run eslint` on both front/server)
- [x] Did you run prettier? (`npm run prettier` on both front/server)
- [ ] If you are adding a new features/services, did you run integration comparator? (`npm run compare-translations` on front)
- [ ] Did you test this pull request in real life? With real devices? If this development is a big feature or a new service, we recommend that you provide a Docker image to [the community](https://community.gladysassistant.com/) for testing before merging.
- [ ] If your changes modify the API (REST or Node.js), did you modify the API documentation? (Documentation is based on comments in code)
- [ ] If you are adding a new features/services which needs explanation, did you modify the user documentation? See [the GitHub repo](https://github.com/GladysAssistant/v4-website) and the [website](https://gladysassistant.com).
- [ ] Did you add fake requests data for the demo mode (`front/src/config/demo.js`) so that the demo website is working without a backend? (if needed) See [https://demo.gladysassistant.com](https://demo.gladysassistant.com).

NOTE: these things are not required to open a PR and can be done afterwards / while the PR is open.

### Description of change

Just adding validation before delete dashboard 

Screenshot 2023-01-21 at 21 11 24
Screenshot 2023-01-21 at 21 10 12

### CODE REVIEW

File: front/src/config/i18n/fr.json [hunk 1]
```diff
@@ -187,6 +187,7 @@
     "editDashboardCancelButton": "Annuler",
     "editDashboardDeleteButton": "Supprimer",
     "editDashboardSaveButton": "Sauvegarder",
+    "editDashboardDeleteText": "Etes-vous sur de vouloir supprimer ce dashboard ?",
```
Comment (Pierre-Gilles) [inline front/src/config/i18n/fr.json]: A few typos + "dashboard" => "tableau de bord" in french :) 

> Êtes-vous sûr de vouloir supprimer ce tableau de bord ?

**Review by Pierre-Gilles — CHANGES_REQUESTED** [review]
Thanks for this PR 🙏

Works fine for me, I just have a small feedback on the french translation

File: front/src/config/i18n/fr.json (hunk 1 above)
Comment (callemand) [reply front/src/config/i18n/fr.json]: Oups :(
Fixed now

**Review by Pierre-Gilles — APPROVED** [review]
Good for me now, thanks for this PR ! 🙏

**Pull request merged**